### DISCUSSION TITLE
Any roadmap planned?

### DISCUSSION THREAD

**Discussion by ZaneDragonBorn** [Swordfish90/Lemuroid#652] [Q&A]
I've noticed we only get updates every month or so, which for a free open source app makes sense, but it does make me partially want to go to platforms like Retroarch due to them having support for Retro Achievements and other functionality. My only issue is, I have to choose between functionality with absolutely terrible UI or Lemuroid with great UI.

Main question is is there a roadmap planned and what features are scheduled to arrive?

I am wanting RetroAchievement support and updates made to the 3ds emulator. Also support for Wii and Gamecube.

**Comment by ghost** [comment]
https://github.com/Swordfish90/Lemuroid/issues/994#issue-2812895959
